src/set_theory/game/position.lean

```diff
@@ -0,0 +1,84 @@
+/-
+Copyright (c) 2020 Fox Thomson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Fox Thomson
+-/
+import set_theory.pgame
+
+/-!
+# Basic definitions about who has a winning stratergy
+
+We define `G.p_position`, `G.n_position`, `G.l_position` and `G.r_position`
+for a pgame `G`, which means the second, first, left and right players
+have a winning stratergy respectivly.
+These are defined by inequalities which can be unfolded with, `pgame.lt_def`
+and `pgame.le_def`.
+-/
+
+namespace pgame
+
+local infix ` ≈ ` := equiv
+
+/-- The player who goes first loses -/
+def p_position (G : pgame) : Prop := G ≤ 0 ∧ 0 ≤ G
+
+/-- The player who goes first wins -/
+def n_position (G : pgame) : Prop := 0 < G ∧ G < 0
```

Comment:
Unless the labels `p` and `n` are standard (not that I remember?), could we have more self-explanatory names? Even `first_loses` rather than `p_position`, etc?